web/src/lib/components/admin-page/settings/job-settings/job-settings.svelte

```diff
@@ -29,7 +28,8 @@
     JobName.Migration,
   ];
 
-  function isSystemConfigJobDto(jobName: JobName): jobName is keyof SystemConfigJobDto {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I'm not much of a Typescript expert. Could this be `unkown` to avoid the `any`?

Reply:
Doesn't look like it:
![image](https://github.com/immich-app/immich/assets/4334196/6aefbe3f-b29d-4c6b-8bc4-6e84856196df)